ClassIntro/Program.cs

```diff
@@ -45,9 +45,7 @@ static void Main(string[] args)
         }
 
     }
-    /// <summary>
```

Comment:
actually you had better add thid summary infos :)

Reply:
I will add summary infos to all objects :)